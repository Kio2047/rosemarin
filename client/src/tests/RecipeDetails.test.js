// import { render, screen } from "@testing-library/react";
// import RecipeDetails from "../components/RecipeDetails";
// import renderWithProvider from "./testUtils/renderWithProvider";
// import { freshLoginTestState } from "./testUtils/testStates"
// import registerIcons from "./testUtils/registerIcons"

// // jest.mock("@fortawesome/react-fontawesome", () => ({
// //   FontAwesomeIcon: ''
// // }));

// registerIcons();

// jest.mock('react-router-dom', () => ({
//   ...jest.requireActual('react-router-dom'),
//   useParams: () => ({
//     id: "8398"
//   }),
// }));

// describe("Recipe Details Component", () => {

//   test("should match the snapshot", () => {
//     const preloadedState = freshLoginTestState;

//     const { container } = renderWithProvider(<RecipeDetails />, {
//       preloadedState
//     });

//     expect(container.firstChild).toMatchSnapshot();

//   })
// });