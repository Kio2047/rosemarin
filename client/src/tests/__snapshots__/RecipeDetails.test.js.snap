// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe Details Component should match the snapshot 1`] = `
<div
  class="h-[300px] flex justify-between items-center"
>
  <div
    class="bg-top-img3 w-full h-full bg-auto bg-no-repeat bg-center bg-cover"
  />
</div>
`;
